packages/clerk-js/src/ui/components/UserButton/useMultisessionActions.tsx

```diff
@@ -12,8 +12,7 @@ type UseMultisessionActionsParams = {
   actionCompleteCallback?: () => void;
   navigateAfterSignOut?: () => any;
   navigateAfterMultiSessionSingleSignOut?: () => any;
```

Comment:
Not related to this PR, but I thought I'd note `navigateAfterMultiSessionSingleSignOut` seems entirely unused and could probably be cleaned up?

Reply:
it is used [here](https://github.com/clerk/javascript/blob/b10755efbda6ce2731188ea8187d55e239506f8b/packages/clerk-js/src/ui/components/UserButton/useMultisessionActions.tsx#L14C3-L14C41)

Reply:
This got my head spinning, until I realized I _put the comment on the wrong line_! 😅

I meant to comment `navigateAfterSwitchSession` which does not seem to be used?

Reply:
I believe the PR already removes it.